### PR TITLE
1 degree coastline buffer

### PR DESCRIPTION
**Pull request by norlandrhagen**
Buffers natural-earth coastline masking file by 2 degrees. 

image

@jhamman @andersy005

### CODE REVIEW

**Comment by norlandrhagen** [comment]
pre-commit.ci autofix

File: cmip6_downscaling/methods/common/utils.py [hunk 1]
```diff
@@ -86,7 +87,17 @@ def subset_dataset(
 
 def apply_land_mask(ds: xr.Dataset) -> xr.Dataset:
     """
-    Apply a land mask to a dataset with lat/lon coordinates
+    Apply a land mask to a dataset with lat/lon coordinates. Land mask is natural_earth_v5_0_0 buffered by 2 degrees.
+
+    --- Regenerate buffer file --
```
Comment (andersy005) [inline cmip6_downscaling/methods/common/utils.py:92]:
```suggestion
    Notes
    --------
    # Regenerate buffer file
```

**Comment by norlandrhagen** [comment]
pre-commit.ci autofix

**Review by andersy005 — APPROVED** [review]

**Comment by norlandrhagen** [comment]
pre-commit.ci autofix

**Pull request merged**